contract/src/offer-up-proposal.js

```diff
@@ -66,8 +69,12 @@ export const startOfferUpContract = async permittedPowers => {
 
   const istIssuer = await istIssuerP;
   const istBrand = await istBrandP;
+  const timerService = await await chainTimerServiceP;
```

Comment:
I don't know whether `await await` breaks anything, but it's at best unnecessary.